.github/workflows/wheels.yml

```diff
@@ -12,12 +12,24 @@ jobs:
   linux:
     name: Linux
     runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        arch: [x86_64, aarch64]
     steps:
     - name: Checkout pygit2
       uses: actions/checkout@v2
 
-    - run: |
-        /bin/sh travis/build-all-manylinux-wheels.sh
+    - name: Set up QEMU
+      if: ${{ matrix.arch == 'aarch64' }}
+      uses: docker/setup-qemu-action@v1
+
+    - name: x86_64 wheels
+      if: ${{ matrix.arch == 'x86_64' }}
+      run: /bin/sh travis/build-all-manylinux-wheels.sh :x86_64
+
+    - name: aarch64 wheels
+      if: ${{ matrix.arch == 'aarch64' }}
+      run: /bin/sh travis/build-all-manylinux-wheels.sh _aarch64
```

Comment:
Please pass `matrix.arch` to `travis/build-all-manylinux-wheels.sh`, then the same step will work for both arches. The idea is to have as much logic as possible in the shell scripts, so they can be easily used from the command line, not just from CI.

For example, better to type:

```
$ sh travis/build-all-manylinux-wheels.sh aarch64
```

Than:

```
$ sh travis/build-all-manylinux-wheels.sh _aarch64
```